src/NServiceBus.Transport.RabbitMQ.TransportTests/NServiceBus.Transport.RabbitMQ.TransportTests.csproj

```diff
@@ -11,10 +11,19 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
-    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="7.1.6" />
+    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="7.1.6" GeneratePathProperty="true" />
     <PackageReference Include="NUnit" Version="3.11.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
     <PackageReference Include="RabbitMQ.Client" Version="5.1.0" />
   </ItemGroup>
 
+  <!-- Replaces transport tests/classes as of 7.1.6. See KTLO issue 218  -->
+  <ItemGroup>
+    <Compile Remove="$(PkgNServiceBus_TransportTests_Sources)\**\When_on_error_throws.cs" />
+    <Compile Remove="$(PkgNServiceBus_TransportTests_Sources)\**\NServiceBusTransportTest.cs" />
+    <Compile Remove="$(PkgNServiceBus_TransportTests_Sources)\**\TransportTestLoggerFactory.cs" />
+    <Compile Include="NServiceBusTransportTest.cs" />
```

Comment:
These three Compile Include lines need to be remove. This is causing the files to be included in the Compile group twice.